mautic/mautic
# Push lead owner to Sugar CRM through forms and campaigns

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | ✓
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Lead owners were not being pushed to Sugar CRM through forms and campaigns. This PR fixes that.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Connect your instance to Sugar CRM
2. Make a form to push leads to Sugar
3. Make a lead with all the necessary data as well as an owner
4. Submit the form with the lead's email
5. Notice the lead has no owner in Sugar

#### Steps to test this PR:
1. Connect your instance to Sugar CRM
2. Make a form to push leads to Sugar
3. Make a lead with all the necessary data as well as an owner
4. Submit the form with the lead's email
5. Notice the lead has an owner in Sugar

#### List deprecations along with the new alternative:
> N/A

#### List backwards compatibility breaks:
> N/A
